### PR TITLE
Stresstest changed to use kuksa_viss_client

### PR DESCRIPTION
**Pull request by linussieweke**
#197

### CODE REVIEW

File: test/stresstest/config.ini [hunk 1]
```diff
@@ -1,4 +1,8 @@
+
 [vss]
-server=ws://localhost:8090
-jwttoken=../../kuksa_certificates/jwt/all-read-write.json.token
-timout=0.1
+
+ip=127.0.0.1
+port=8090
+insecure=true
```
Comment (wenwenchenbosch) [inline test/stresstest/config.ini:6]: insecure should not be necessary any more. can you test it

File: test/stresstest/stressClient.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+    def getConfig(self):
+
+        parser = argparse.ArgumentParser()
+
+        parser.add_argument("-s", "--server", help="VSS server", type=str)
+        parser.add_argument("-j", "--jwt", help="JWT security token file", type=str)
+        parser.add_argument("-t", "--timeout", help="Timeout between requests", type=float)
+
+        args = parser.parse_args()
```
Comment (wenwenchenbosch) [inline test/stresstest/stressClient.py:22]: args hier for argument parser does not necessary any more richtig?

File: test/stresstest/stressClient.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        self.cfg['timeout'] = vsscfg.getfloat("timeout", 0.1)
+
+    def connect(self,cfg):
+        insecure = cfg['insecure']
+        """Connect to the VISS Server"""
+        if hasattr(self, "commThread"):
+            if self.commThread != None:
+                self.commThread.stop()
+                self.commThread = None
+        config = {'ip':self.cfg['ip'],
```
Comment (wenwenchenbosch) [inline test/stresstest/stressClient.py:45]: `config` hier is useless

File: test/stresstest/stressClient.py [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    cfg = {}
+
+
+    def getConfig(self):
+
+        config = configparser.ConfigParser()
+        config.read('config.ini')
+
+        vsscfg = config['vss']      #get data from config file
+        """self.cfg['ip'] = vsscfg.get("ip")
```
Comment (wenwenchenbosch) [inline test/stresstest/stressClient.py:22]:
```suggestion
        self.cfg['ip'] = vsscfg.get("ip")
```

**Review by wenwenchenbosch — CHANGES_REQUESTED** [review]

File: test/stresstest/stressClient.py (hunk 4 above)
Comment (linussieweke) [reply test/stresstest/stressClient.py:22]: Agreed, forgot to commit cleanup...

**Review by wenwenchenbosch — APPROVED** [review]

**Pull request merged**